src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -95,7 +95,7 @@ def save_pretrained_wrapper(
                     "skip_compression_stats=True"
                 )
                 sparsity_config = SparsityConfigMetadata.from_pretrained(
-                    model, state_dict=state_dict, compress=False
+                    model, state_dict=state_dict, compress=save_compressed
                 )
```

Comment:
Do you think we should change `save_compressed: bool = True` to False in order to preserve previous behavior?